src/commonTest/kotlin/it/krzeminski/snakeyaml/engine/kmp/issues/issue47/SetWidthTest.kt

```diff
@@ -0,0 +1,78 @@
+package it.krzeminski.snakeyaml.engine.kmp.issues.issue47
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.shouldBe
+import it.krzeminski.snakeyaml.engine.kmp.api.Dump
+import it.krzeminski.snakeyaml.engine.kmp.api.DumpSettings
+import it.krzeminski.snakeyaml.engine.kmp.api.Load
+import it.krzeminski.snakeyaml.engine.kmp.api.LoadSettings
+import it.krzeminski.snakeyaml.engine.kmp.api.StringStreamDataWriter
+import it.krzeminski.snakeyaml.engine.kmp.common.ScalarStyle
+import it.krzeminski.snakeyaml.engine.kmp.emitter.Emitter
+import it.krzeminski.snakeyaml.engine.kmp.events.*
+
+class SetWidthTest : FunSpec({
```

Comment:
suggestion: I think we need to make test names a bit more explicit. Right now (at least for me) it is hard to understand what test does with looking though the whole test method. E.g: 

"emit plain text and split" -> "emitted plain text with limited width should not be split" (or something similar)

Reply:
I'm thinking about writing down some guidelines on porting stuff from snakeyaml-engine, I. e. what things we left untouched, and what we'd like to improve.

Reply:
Sounds like a good idea